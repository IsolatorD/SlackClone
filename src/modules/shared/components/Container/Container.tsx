import React from "react";
import { SafeAreaView, ViewProps } from "react-native";

export interface IContainerProps extends ViewProps {}

export default function Container(props: IContainerProps) {
  return (
    <SafeAreaView
      {...props}
    >
      {props.children}
    </SafeAreaView>
  );
}