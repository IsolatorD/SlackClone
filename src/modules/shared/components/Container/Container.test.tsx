import 'react-native';
import React from 'react';
import Container from './Container';
import { render } from '@testing-library/react-native'

it('Container.tsx shared component render correctly', () => {
  const { toJSON } = render(<Container />);
  expect(toJSON()).toMatchSnapshot();
});
