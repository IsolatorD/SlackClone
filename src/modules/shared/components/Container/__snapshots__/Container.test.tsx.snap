// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container.tsx shared component render correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
/>
`;
